src/main/java/tools/map/making/MapPropertyWrapper.java

```diff
@@ -40,14 +40,12 @@
  * @param <T>
  *        parameters can be: Boolean, String, Integer, Double, Color, File, Collection, Map
  */
-@SuppressWarnings("unchecked")
 public class MapPropertyWrapper<T> extends AEditableProperty {
   private static final long serialVersionUID = 6406798101396215624L;
-  private IEditableProperty property;
+  private final IEditableProperty property;
   // private final Class m_clazz;
   // private final T m_defaultValue;
   private final Method setter;
-  @SuppressWarnings("unused")
```

Comment:
This field actually is unused.  We just kept it to preserve serialization compatibility.

Reply:
Interesting, Intellij sees the assignment and does not mark it as used. I think the proper thing to do is to actually make it unused, ie: remove the assignment, and then comment that we are keeping it around for compat.

Reply:
"Unused," at least from the Eclipse static analyzer's perspective, means "never read."  Regardless, I agree with your proposal.